editor/svg-editor.js

```diff
@@ -5759,20 +5748,21 @@ editor.init = function () {
        */
       setAll () {
         const flyouts = {};
+        const keyHandler = {}; // will contain the action for each pressed key
 
-        $.each(toolButtons, function (i, opts) {
+        toolButtons.forEach((opts) => {
           // Bind function to button
           let btn;
           if (opts.sel) {
-            btn = $(opts.sel);
-            if (!btn.length) { return true; } // Skip if markup does not exist
+            btn = document.querySelector(opts.sel);
```

Comment:
I think this really needs to be in a utility. If you import my Jamilih library (already a part of the project elsewhere), it exports `$` and `$$` for querySelector and querySelectorAll (which you could rename to avoid jQuery conflicts).

Reply:
I need to go deeper into your library but, for now,  I like the idea that every developper would quickly understand what the line does so  I'd rather keep native functions at least until we get rid of jQuery.

Reply:
In Jamilih, it is implemented with just these lines:

```js
const $ = (sel) => doc.querySelector(sel);
const $$ = (sel) => [...doc.querySelectorAll(sel)];
```

However, you are most welcome to add copies of these locally into our utilities file so they are more immediately traceable as to their implementation. But I think it is too frequent of a need not to have it as a utility in *some* form, and having `querySelector` explicit everywhere, while semantic, is just too verbose and distracting imo for the ever-present need for finding items by selector. Developers can follow the import for this to find its implementation as with other functions in the library. For pretty much any other function I agree it is better to directly use native functions when not significantly more verbose.

Reply:
OK, I agree to do this but when we will have no more jQuery in the project so $ can be used without the risk to mix with jQuery $

Reply:
Ok... how does `$q` and `$qq` sound?

Reply:
Ok for me.